Earthfile

```diff
@@ -42,23 +46,28 @@ run-coverage:
     DO rust-udc+CARGO --args="llvm-cov report --html --output-dir /tmp/coverage/"
     SAVE ARTIFACT /tmp/coverage/*
 
+# build runs tests and then creates a release build
 build:
     BUILD +run-tests
     BUILD +build-release
 
+# build-create creates a .crate file for distribution of source code
 build-crate:
     FROM +copy-src
     DO rust-udc+CARGO --args="package"
     SAVE ARTIFACT target/package/*.crate /package/ AS LOCAL artifacts/crate/
 
+# lint runs cargo clippy on the source code
 lint:
     FROM +copy-src
     DO rust-udc+CARGO --args="clippy --all-features --all-targets -- -D warnings"
 
+# fmt formats Rust code according to style guidelines
```

Comment:
```suggestion
# fmt checks whether Rust code is formatted according to style guidelines
```
This actually does not format the code. It only checks whether format is good.

Reply:
Thanks, fixed!